modules/tinymce/src/plugins/directionality/test/ts/browser/DirectionalitySanityTest.ts

```diff
@@ -47,4 +47,139 @@ describe('browser.tinymce.plugins.directionality.DirectionalitySanityTest', () =
       });
     }));
   });
+
+  it('should set two paragraphs to rtl and ltl', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>foo</p><p>bar</p>');
+    TinySelections.setSelection(editor, [ 0 ], 0, [ 1 ], 1);
+    TinyUiActions.clickOnToolbar(editor, 'button[title="Right to left"]');
+    TinyAssertions.assertContentStructure(editor, ApproxStructure.build((s, str) => {
+      return s.element('body', {
```

Comment:
I know the existing tests were using `assertContentStructure` but I think a plain `assertContent` here will be nicer and easier to read. I think we should assert the entire editor content given how simple the output would be in this case

`TinyAssertions.assertContent(editor, '<p dir="rtl">foo</p><p dir="rtl">bar</p>');`

You may want to set `indent: false` in the editor settings so that whitespace isn't added during serialization when `getContent` is called in these assertions.

Reply:
For simple tests I changed to `assertContent` for long content I'm still using `assertContentStructure`.